docs/roadmap.md

```diff
@@ -42,13 +42,13 @@ For highly used services, a common pattern is to deploy the production version t
 
 Many companies in regulated industries are wary of the public cloud and so wish to run all services used by their in house engineers themselves. Companies in this situation wishing to use Humanitec would, therefore, require an on-premise version of Humanitec. Being able to deploy and use Humanitec in this configuration is not particularly complex, however, the support process around it and the pace of receiving updates is the biggest challenge.
 
-### Team Performance Monitoring
+### Single Sign-On (SSO)
 
-Humanitec is sitting on a large amount of data and so has the potential to provide valuable insights about team performance. It would be relatively straightforward to measure time from “commit to production” for example.
+Many companies are using Single Sign-On (SSO) to centrally govern and control system and application access. Humanitec will start providing a SSO solution (most likely based on SAML) on an organizational level to support system admins in their work.
```

Comment:
Many companies use Single Sign-On (SSO) to centrally govern and control system and application access. Humanitec will integrate with SSO solutions, most likely starting with SAML. This will allow organizations manage access to Humanitec using their existing tools and systems.

Reply:
Thx